### ISSUE TITLE
Bug with InlineQueryResultCachedSticker + InlineQueryResultArticle

### ISSUE THREAD

**Issue by DO97** [TelegramMessenger/Telegram-iOS#122]
This is a missing feature of Telegram for iOS and probably macOS.
See the difference:
> ![IMG_20190817_145112_229](https://user-images.githubusercontent.com/17525578/63212210-26b47700-c101-11e9-9f3f-e71b05dbea7f.jpg)
Android

> ![IMG_20190817_145116_371](https://user-images.githubusercontent.com/17525578/63212221-4a77bd00-c101-11e9-8c61-efc8ee3382ef.jpg)
iOS

In my code, I mixed two types of inline queries: the one that shows stickers and the last line you can see in the screenshots (InlineQueryResultCachedSticker and InlineQueryResultArticle) .
With "mixed" I mean that I added IQRCS and IQRA separately, but called both into one result.

It seems that it is supported (without any issue) only on Android systems and Telegram Desktop/Web.

**Comment by laktyushin** [comment]
Could you provide an example bot with steps to reproduce?

**Comment by DO97** [comment]
> Could you provide an example bot with steps to reproduce?

Probably no, but I give you a practical example:

def add_other_cards_team(player, results, game):
    """Add partner hand cards"""
    partner = player.next.next

    results.append(
        InlineQueryResultArticle(
            "team_hand",
            thumb_url=_("http://gamesarena.altervista.org/UnoGABot/Heart.jpg"),
            title=_("Carta di {name}",
                    "Carte di {name}",
                    len(partner.cards))
                    .format(name=partner.user.first_name),
            description=', '.join([repr(card) for card in partner.cards]),
            input_message_content=game_info(game)
        )
    )

def add_card(game, card, results, can_play):
    """Add an option that represents a card"""

    if can_play:
        results.append(
            Sticker(str(card), sticker_file_id=c.STICKERS[str(card)])
        )
    else:
        results.append(
            Sticker(str(uuid4()), sticker_file_id=c.STICKERS_GREY[str(card)],
                    input_message_content=game_info(game))
        )


Here I mix the results:

**Comment by DO97** [comment]
> Could you provide an example bot with steps to reproduce?

> Could you provide an example bot with steps to reproduce?

Probably no, but I give you a practical example:

```
from telegram import ParseMode, InlineQueryResultArticle, InputTextMessageContent, \
    InlineQueryResultCachedSticker as Sticker

def add_other_cards_team(player, results, game):
    """Add partner hand cards"""
    partner = player.next.next

    results.append(
        InlineQueryResultArticle(
            "team_hand",
            thumb_url=_("http://www.sabethahospital.com/pictures/content/198211.jpg"),
            title=_("Carta di {name}",
                    "Carte di {name}",
                    len(partner.cards))
                    .format(name=partner.user.first_name),
            description=', '.join([repr(card) for card in partner.cards]),
            input_message_content=game_info(game)
        )
    )


def add_card(game, card, results, can_play):
    """Add an option that represents a card"""

    if can_play:
        results.append(
            Sticker(str(card), sticker_file_id=c.STICKERS[str(card)])
        )
    else:
        results.append(
            Sticker(str(uuid4()), sticker_file_id=c.STICKERS_GREY[str(card)],
                    input_message_content=game_info(game))
        )
```

Here (in another file) I mix the results:
```
                # 1st result
                for card in sorted(player.cards):
                    add_card(game, card, results,
                             can_play=(card in playable and
                                            str(card) not in added_ids))
                    added_ids.append(str(card))

                # Rightly excluded
                if game.mode != 'Team':
                    add_gameinfo(game, results) # Not included

                # 2nd result
                if game.mode == 'Team':
                    add_other_cards_team(player, results, game)
```
Basically, it works when you tap on the empty boxes but it doesn't show stickers as on Android.

**Comment by DO97** [comment]
@laktyushin

**Comment by DO97** [comment]
@laktyushin Do you need any other information?

**Comment by DO97** [comment]
Not fixed in 5.11 version

**Comment by DO97** [comment]
> Could you provide an example bot with steps to reproduce?

Sorry, if you mean the bot referenced in the screenshots, then you can try it in: https://t.me/joinchat/DXQ0RkJ_uCiax22Z58m5Aw
Or
https://t.me/joinchat/DXQ0RlVJyD4EcGiocKedfw

**Comment by laktyushin** [comment]
Fixed in 5.11.1
